CHANGELOG.md

```diff
@@ -4,8 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.3] - 2024-09-09
 
-## [Unreleased]
+### Fixed
```

Comment:
So I think this should be:

```markdown
### Changed
- Upgrade Iguana for PHP 8 compatibility

### Fixed
- Accordion ACF warning that ACF now automatically escapes unsafe HTML
```

Did the Iguana upgrade happen in a different PR?

Reply:
> So I think this should be:
> 
> ```
> ### Changed
> - Upgrade Iguana for PHP 8 compatibility
> 
> ### Fixed
> - Accordion ACF warning that ACF now automatically escapes unsafe HTML
> ```
> 
> Did the Iguana upgrade happen in a different PR?

@snim2 yes, one made by Patrick